TesseractCore/Identifier.swift

```diff
@@ -5,7 +5,11 @@ public struct Identifier: Comparable, Hashable, Printable {
 		self.value = Identifier.cursor++
 	}
 
-
+    internal init(value: Int) {
+        self.value = value
+        Identifier.cursor = value
+    }
+    
```

Comment:
I don’t think we ought to reset `cursor`, or at least not like this:

1. At a minimum, it should be the maximum of `cursor` and `value + 1`.
2. Creating a graph, loading another graph, and then adding another node to the first graph will cause collisions if we do that.

Ultimately this `Identifier` type was intentionally short-sighted, sort of a minimum viable identifier. It seems it’s less viable now, but that can probably be addressed orthogonally to importing (so don’t worry about that for now).